### DISCUSSION TITLE
Edge function

### DISCUSSION THREAD

**Discussion by piyush672** [supabase/supabase#22488] [Questions]
Hi till yesterday i was able to invoke a edge function but today i am only getting server error i.e 500 so is it happening with others too?

**Comment by GaryAustin1** [comment]
Not sure if you are one of the users here:  https://discord.com/channels/839993398554656828/1006358244786196510/threads/1225185388625264721
There is comment re-posted form support there.  A couple of 500 reports, but not clear all are the same.
